sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusModelFactory.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Messaging.ServiceBus.Primitives
+{
+    /// <summary>
+    /// This class contains methods to create certain ServiceBus models.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class ServiceBusModelFactory
+    {
+        /// <summary>
+        /// Creates a new ServiceBusReceivedMessage instance for mocking.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static ServiceBusReceivedMessage ServiceBusReceivedMessage(
```

Comment:
I didn't really know what to do for testing here. I can add a unit test, but it would basically just check to make sure the properties are set as expected. Didn't know if that would provide value or not.

Reply:
Yeah a unit test would be nice just to make sure that the properties are correctly set as you say.

Reply:
Added a test to check default creation and all parameters being used.